terms/inputType.jsonld

```diff
@@ -0,0 +1,9 @@
+{
+    "@context": {
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#"
+    },
+    "@id": "inputType",
+    "@type": "rdfs:Class",
```

Comment:
this should be a property since it takes on a value.  see: https://schema.org/additionalName.jsonld for an example.

Reply:
this should also include rangeincludes and domainincludes